.github/workflows/mvn-verify-check.yml

```diff
@@ -95,3 +97,33 @@ jobs:
           -DskipTests
           -Dskip
           -Dmaven.javadoc.skip
+
+  verify-modules-with-jdk11:
+    needs: get-noSnapshot-versions-from-dist
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        version: ${{ needs.get-noSnapshot-versions-from-dist.outputs.sparkJDK11Versions }}
```

Comment:
`[mvn[compile,RAT,scalastyle,docgen]: .github#L1](https://github.com/NVIDIA/spark-rapids/pull/6901/files#annotation_5406717646)
Error when evaluating 'strategy' for job 'verify-modules-with-jdk11'. .github/workflows/mvn-verify-check.yml (Line: 106, Col: 18): Unexpected type of value '312', expected type: Sequence.`

failed typing https://github.com/NVIDIA/spark-rapids/actions/runs/3328337589/attempts/1